cspt/config.py

```diff
@@ -2,11 +2,11 @@
 from datetime import date as dt
 from datetime import timedelta
 # UPDATE: update this each semester
-REPO = 'http://compsys-progtools.github.io/fall024/'
-BASE_URL = 'https://raw.githubusercontent.com/compsys-progtools/fall2024/main/_'
+REPO = 'http://compsys-progtools.github.io/spring2025/'
+BASE_URL = 'https://raw.githubusercontent.com/compsys-progtools/spring2025/main/_'
 
 GH_APPROVERS = ['brownsarahm','marcinpawlukiewicz','VioletVex']
-EARLY_BIRD_DEADLINE= datetime.fromisoformat('2024-09-20')
+EARLY_BIRD_DEADLINE= datetime.fromisoformat('2025-01-30')
```

Comment:
normally I give this bonus for about 3 weeks, this is a little extra incentive for students to get work done right from the start. You can keep this date if you prefer, but I wanted to share my thinking